deploy/helm/sumologic/templates/events-deployment.yaml

```diff
@@ -17,6 +17,14 @@ spec:
         {{ template "sumologic.labels.common" . }}
     spec:
       serviceAccountName: {{ template "sumologic.fullname" . }}
+{{- if .Values.eventsDeployment.nodeSelector }}
+      nodeSelector:
+{{ toYaml .Values.eventsDeployment.nodeSelector | indent 8 }}
+{{- end -}}
+{{- if .Values.deployment.tolerations }}
```

Comment:
this should be `.Values. eventsDeployment.tolerations`

Reply:
Good with that...

Reply:
Fixed